baybe/acquisition/base.py

```diff
@@ -39,15 +39,19 @@ def to_botorch(
         self,
         surrogate: Surrogate,
         searchspace: SearchSpace,
-        train_x: pd.DataFrame,
-        train_y: pd.DataFrame,
+        measurements: pd.DataFrame,
     ):
         """Create the botorch-ready representation of the function."""
         import botorch.acquisition as botorch_acqf_module
 
         acqf_cls = getattr(botorch_acqf_module, self.__class__.__name__)
         params_dict = filter_attributes(object=self, callable_=acqf_cls.__init__)
 
+        train_x, train_y = (
+            surrogate.transform_inputs(measurements),
+            surrogate.transform_targets(measurements),
+        )
```

Comment:
why this weird tuple notation? Now its 4 instead of 2 lines

Reply:
If you don't like the above, you gonna certainly hate this one here 😄 
```python
        ((train_x, train_y),) = (
            to_tensor(
                self.transform_inputs(measurements),
                self.transform_targets(measurements),
            ),
        )
```
Seriously, no idea how that happened, I think combination auf autoformat and copying certain expressions like `train_x, train_y` or similar. Will fix and close once resolved

Reply:
Ah, now I remember. Because I copied the code from surrogate, which additionally has the `to_tensor` that I then removed:
<img width="342" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/642c8eae-718b-4970-b3c6-42c9435168a9">
Do you want me to split that part up as well?

Reply:
i saw that as I also wanted to complain there but its a different story, there I dont mind either going with 2 separate lines or the tuple, but here?

Reply:
ok, will keep the tuple for `to_tensor` but split it in the other place 👍🏼

Reply:
changed